### PR TITLE
Change LDAP references to AD/LDAP in docs

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

**Comment by jasonblais** [comment]
This PR is against #448

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -613,25 +613,25 @@ ________
 
 AD/LDAP (Enterprise)
 ```````````````````````````
-Enable sign-in with LDAP
+Enable sign-in with AD/LDAP
```
Comment (lfbrock) [inline source/administration/config-settings.rst]: Are we changing the actual System Console settings to say AD/LDAP as well? 

I know we changed the title of the page, but wasn't sure if the settings on the page also got renamed

Comment (jasonblais) [inline source/administration/config-settings.rst]: Yep, per https://github.com/mattermost/platform/pull/3925 and https://mattermost.atlassian.net/browse/PLT-3766

**Comment by esethna** [comment]
+1

**Comment by lfbrock** [comment]
+1

**Pull request merged**